### DISCUSSION TITLE
Tagged Templates don't rerun

### DISCUSSION THREAD

**Discussion by waryu-YND** [solidjs/solid#2074] [Q&A]
Hello,

I've run into an issue with Solid.js where tagged templates don't seem to rerun on signal updates. Here's a brief example:
```tsx
import { render } from "solid-js/web";
import { createSignal } from "solid-js";

function App() {
  const [count, setCount] = createSignal(0);

  const unitCount = (templates: string[], ...literals: string[]) => {
    return count() + templates[0]
  }

  setInterval(()=>{
    setCount((now) => now + 1);
  }, 1000);

  return (
    <div>
      <p>{unitCount(["s"])}</p> // Updates every second
      <p>{unitCount`s`}</p> // Does not update after initial render
    </div>
  )
}

render(() => <App />, document.getElementById("app")!);
```
The first ``<p>{unitCount(["s"])}</p>`` updates as expected, but ``<p>{unitCount`s`}</p>`` does not update after the initial render. Is this expected behavior in Solid.js? How can I get tagged templates to react to signal updates?

**Comment by barvian** [comment]
I'm not sure if this is expected behavior or not, but you could work around it [like this](https://playground.solidjs.com/anonymous/71d467b3-bf20-4568-81d4-95c5ae706957)

**Comment by ryansolid** [comment]
Ok.. yeah I don't think this isn't in our heuristic... can you report this as an issue? We should see tagged template literals as function calls. So JSX isn't wrapping the expression.
